easybuild/tools/package/utilities.py

```diff
@@ -105,51 +123,37 @@ def package_with_fpm(easyblock):
 
     _log.debug("The dependencies to be added to the package are: %s",
                pprint.pformat([easyblock.toolchain.as_dict()] + easyblock.cfg.dependencies()))
-    depstring = ''
+    deplist = []
     for dep in deps:
         if dep.get('external_module', False):
             _log.debug("Skipping dep marked as external module: %s", dep['name'])
         else:
             _log.debug("The dep added looks like %s ", dep)
             dep_pkgname = package_naming_scheme.name(dep)
-            depstring += " --depends %s" % quote_str(dep_pkgname)
+            deplist.extend(["--depends", dep_pkgname])
```

Comment:
It just kept it more like it was, but sure I can do it the other way now too since I have the cmdlist moved up.